.scripts/_tmux.sh

```
@@ -0,0 +1,259 @@
+  
+  # Only start a new session if not already in tmux
+  if [ ! -z "$TMUX" ]; then return 0; fi
+
+  echo:Tmux "Starting a new tmux session: $session_name"
+  
+  # Set environment variables to track that we started tmux
+  export TMUX_STARTED_BY_SCRIPT=1
+  export TMUX_SESSION_NAME="$session_name"
+  
+  # Start new session and execute the current script inside it
+  echo:Tmux "exec tmux new-session -s \"$session_name\" \"$0\" $@"
```

Shellcheck (0.10.0)

[error] Argument mixes string and array. Use * or separate argument.

(SC2145)